packages/waku/src/lib/handlers/handler-dev.ts

```diff
@@ -154,12 +154,17 @@ export function createHandler<
     }
     if (ssr) {
       try {
+        const mainJsPath = joinPath(
+          config.basePath,
+          config.srcDir,
+          config.mainJs,
+        );
         const readable = await renderHtml({
           config,
           pathname: req.url.pathname,
           searchParams: req.url.searchParams,
           htmlHead: `${config.htmlHead}
-<script src="${config.basePath}${config.srcDir}/${config.mainJs}" async type="module"></script>`,
```

Comment:
Was this causing any problem? `joinPath` isn't specifically designed for URLs.

Reply:
No, I just wanna saw if this cause the error